package.json

```diff
@@ -51,6 +51,7 @@
     "abortcontroller-polyfill": "^1.4.0",
     "aes-js": "^3.1.2",
     "alif-toolkit": "^1.2.9",
+    "all": "0.0.0",
```

Comment:
Looks like you may have accidentally added this dependency? Could you delete this line?

Reply:
Thank you! I fixed the error

Reply:
Thanks! Looks like your text editor changed the indentations? Would you mind reseting the file?

Reply:
Sorry about that, just fixed it!